dask_bigquery/__init__.py

```diff
@@ -1 +1,3 @@
 from .core import read_gbq
+
+__version__ = "0.0.1"
```

Comment:
To keep this in sync with `setup.py`, you have a few options. For the `googleapis` packages, we've started using a separate `version.py` file, which is easier to extract the version string from.

https://github.com/googleapis/python-api-core/blob/083e6e92ac870f2ab0bfcbae83fd0f30fe013eb7/setup.py#L50-L53

Reply:
Thanks for the suggestion @tswast! Eventually we'll probably move to using [`versioneer`](https://github.com/python-versioneer/python-versioneer) for managing the package version number. That'll make `dask-bigquery` consistent with most other Dask-related packages.